scriptmodules/emulators/pifba.sh

```diff
@@ -62,6 +63,7 @@ function configure_pifba() {
 
     local def=0
     isPlatform "rpi1" && def=1
+    addSystem $def "$md_id" "arcade" "$md_inst/fba2x %ROM%"
```

Comment:
instead of $def - 0 would be better as we don't want either of the fba emulators as default for arcade. I think no default should be set - the user will be asked then on first run by runcommand.cfg